src/server/helpers/utils.js

```diff
@@ -174,8 +175,7 @@ export function incrementEditorEditCountById(orm, id, transacting) {
  * @returns {Promise} a promise which will be fulfilled when the operation to
  *                    truncate tables completes
  */
-export function truncateTables(Bookshelf, tables) {
-	return Promise.each(
```

Comment:
What's the reason for this change? I have a feeling "each" is needed for its sequential execution of promises, since the tables are often passed in in an order that minimises dependencies.